### ISSUE TITLE
Prior Knowledge Given to findGraphMulti

### ISSUE THREAD

**Issue by gabriel-ruiz** [ysamwang/highDNG#1]
Hi, 

Regarding Section _4.3 Pre-selection of neighborhoods_ in [1] and the corresponding example (I think) in `[...]/tests/highD_preSelect.R` [2]: what is the format of `skel` in
`findGraphMulti(Y, maxInDegree = maxInDegree, cutOffScaling = cs, B = skel, degree = deg,
                                                   verbose = T))`?

For example, if I run neighborhood lasso regression to get the support for an undirected graph, should `skel` be the corresponding p by p matrix with 1's and 0's? 

[1] Y. S. Wang and M. Drton. "High-dimensional causal discovery under non-Gaussianity." _Biometrika_ (2020). 
[2] https://github.com/ysamwang/highDNG/blob/master/highDlingam/tests/highD_preSelect.R

Thank you for posting this code. It has worked great so far.

**Comment by ysamwang** [comment]
Yes! It should be 1's (indicating possible edge) and 0's (indicating no edge) with all diagonal elements also being 0

**Comment by gabriel-ruiz** [comment]
Awesome, thank you!
